databases/week3/Menu.sql

```diff
@@ -0,0 +1,462 @@
+WHERE
+  id = 3;
+
+-- Update a reservation with any id
+UPDATE
+  Reservation
+SET
+  contact_name = 'Victor'
+WHERE
+  id = 1;
+
+-- Delete a reservation with any id
+DELETE FROM
+  Reservation
+WHERE
+  id = 4;
+
+-- Queries to write on Review
+  -- Get all reviews
+SELECT
+  *
+FROM
+  Review;
+
+-- Add a new review
+INSERT INTO
+  `Review`(
+    `title`,
+    `description`,
+    `meal_id`,
+    `stars`,
+    `created_date`
+  )
+VALUES
+  (
+    "Very Good",
+    "Taste is so good",
+    5,
+    4,
+    "2022-02-05"
+  );
+
+-- Get a review with any id
+SELECT
+  *
+FROM
+  Review
+WHERE
+  id = 2;
+
+-- Update a review with any id
+UPDATE
+  Review
+SET
+  stars = 5
+WHERE
+  id = 3;
+
+-- Delete a review with any id
+DELETE FROM
+  Review
+WHERE
+  id = 2;
+
+-- Add different meals, reservations and reviews
+INSERT INTO
+  `Meal` (
+    `title`,
+    `description`,
+    `location`,
+    `when`,
+    `max_reservations`,
+    `price`,
+    `created_date`
+  )
+VALUES
+  (
+    "Chicken Fry",
+    "grilled chicken with tikka masala",
+    "USA",
+    "2022-01-25 10:00:00",
+    60,
+    100,
+    "2022-01-20"
+  ),
+  (
+    "Baked Honey Mustard Chicken",
+    "honey mustard and onion sauce with pepper salt",
+    "Copenhagen",
+    "2022-02-18 12:00:00",
+    30,
+    150,
+    "2022-02-10"
+  );
+INSERT INTO
+  `Reservation` (
+    `number_of_guests`,
+    `meal_id`,
+    `created_date`,
+    `contact_phonenumber`,
+    `contact_name`,
+    `contact_email`
+  )
+VALUES
+  (
+    20,
+    6,
+    "2022-01-20",
+    "23234466",
+    "Mikkel",
+    "mikkel@hotmail.com"
+  ),
+  (
+    45,
+    7,
+    "2022-03-05",
+    "75755050",
+    "Adams",
+    "adams@gmail.com"
+  );
+INSERT INTO
+  Review (
+    `title`,
+    `description`,
+    `meal_id`,
+    `stars`,
+    `created_date`
+  )
+VALUES
+  (
+    "Bad",
+    "Taste is so bad",
+    6,
+    1,
+    "2022-03-04"
+  ),
+  (
+    "Good",
+    "Taste is ok but needs to improve",
+    7,
+    3,
+    "2022-03-02"
+  );
+
+-- Additional queries
+  -- Get meals that has a price smaller than a specific price
+SELECT
+  *
+FROM
+  Meal
+WHERE
+  price < 90;
+
+-- Get meals that still has available reservations
+SELECT
+  Meal.title,
+  Reservation.number_of_guests,
+  Meal.max_reservations
+FROM
+  Meal
+  JOIN Reservation ON Meal.id = Reservation.meal_id
+WHERE
+  Meal.max_reservations > Reservation.number_of_guests;
+
+-- Get meals that partially match a title.
+SELECT
+  *
+FROM
+  Meal
+WHERE
+  title LIKE '%Chicken%';
+
+-- Get meals that has been created between two dates
+SELECT
+  *
+FROM
+  Meal
+WHERE
+  created_date BETWEEN "2022-02-20"
+  AND "2022-03-20";
+
+-- Get only specific number of meals fx.return only 5 meals
+SELECT
+  *
+FROM
+  Meal
+LIMIT
+  5;
+
+-- Get the meals that have good reviews
+SELECT
+  Meal.title
+FROM
+  Meal
+  JOIN Review ON Meal.id = Review.meal_id
+WHERE
+  Review.stars > 4;
+
+-- Get reservations for a specific meal sorted by created_date
+SELECT
```

Comment:
Good! It would be nice with a few more entries in the database so we could see the effect of ordering the date